data/human/human missions.txt

```diff
@@ -7453,3 +7453,405 @@ mission "Adelita 5"
+			variant
+				"Wasp" 3
+			variant
+				"Quicksilver"
+
+	on complete
+		conversation
+			`As soon as you land, you see a message from Howard:`
+			`	"<first> <last>! Looks like I beat you landing on <planet>! Meet me at my office near the spaceport to talk further. (Again, note the address change.)"`
+			`		Howard Vendeur`
+			`		Investor Relations Manager, Megaparsec Incorporated`
+
+
+mission "Syndicate Business 5"
+	name "Disable a Marauder ship on <planet>"
+	description "Howard wants you to disable a Marauder ship in exchange for an investment opportunity."
+	priority
+	to offer
+		has "Syndicate Business 4: done"
+	source "Sunracer"
+	waypoint "Oblate"
+	destination "Sunracer"
+	on offer
+		conversation
+			`Even amidst the mix of landing pads, hangars, and cranes, it's hard to miss the Megaparsec corporate office. After a short walk, you reach the towering building. The lobby is sleek and modern, filled with the quiet hum of efficiency. To your surprise the receptionist waves you on and you take an elevator up to the eighth floor, where Howard awaits.`
+			`	The office reminds you of the one you were in on Hephaestus, though more understated. Howard's office is a modest yet modern space, featuring a smart desk, a few sleek chairs, and large windows overlooking the busy shipyards. You can't help noticing that, like the Hephaestus office, the minimalist decor hints more at style than substance.`
+			`	"Glad you could make it," Howard says, extending his hand. "Apologies for the inconvenience, but it's better to discuss these matters away from potentially 'interested' parties." He motions for you to join him at one of the few sleek chairs in the center of the room.`
+			choice
+				`	(Take a seat.)`
+				`	(Remain standing.)`
+					goto shrug
+			`	Howard sits beside you in an oddly intimate expression.`
+				goto familiar
+			label shrug
+			`	Howard shrugs and takes a seat anyway.`
+			label familiar
+			`	Once settled, he leans back in his chair and gives you a serious look. "Tell me," he says, "are you familiar with Marauder ships?"`
+			choice
+				`	"Yes, of course."`
+					goto ask
+				`	"No, I can't say I am."`
+			`	Howard raises an eyebrow. "Marauder ships are nasty pieces of work," he explains. "They're ships with numerous modifications - mostly with combat in mind. More shields, weapons, engines, and outfit space, with enough firepower to tear through a lot of defenses before anyone knows what hit them. They're often used by rogue groups that operate outside the law, preying on vulnerable targets."`
+			label ask
+			`	Howard pauses for a moment before continuing. "Megaparsec is interested in getting our hands on one. We want to study its design, see what makes it tick. We've received word that a Marauder ship was recently spotted in Oblate. I need you to track it down and disable it. Once it's crippled, our recovery team will handle the rest."`
+			`	He gives you a pointed look. "Think you can handle that?"`
```

Comment:
Somehow I associate "pointed" as if he's secretly wondering if he can trust you.
Is that just me, or should you add a `to display` to make this only show if you've been mean to him, and use a different word if you haven't?

Reply:
Great idea

Reply:
And done